packages/core/src/sdk/deliveryPromise/useDeliveryPromise.ts

```diff
@@ -473,10 +503,14 @@ export function useDeliveryPromise({
     facets,
     onPostalCodeChange,
     onDeliveryFacetChange,
-    deliveryLabel:
-      deliveryPromiseSettings?.deliveryMethods?.title ?? 'Delivery',
-    deliveryOptionsLabel:
-      deliveryPromiseSettings?.deliveryOptions?.title ?? 'Delivery Option',
+    labelsMap: {
```

Comment:
I've made this change to make it easier to access those labels since the list is getting bigger, if you think it's a problem since `deliveryLabel` is already in main, I can change it :/

Reply:
Actually I liked this approach 🙈 
I don't see a problem now, better to change now than later. Do you think we will need to add more items to this list? Maybe we could extract this to a const and call it on the return, a little bit cleaner.